web/src/main/resources/static/js/tree.js

```diff
@@ -63,67 +123,18 @@ var tree = (function () {
             .append("svg:g")
             .attr("transform", "translate(" + m[3] + "," + 300 + ")");
 
-        d3.tsv(txtUrl, function (csv) {
-            var rootDatum = new UniqueTreeNodeDatum();
-            rootDatum.name = 'Tissue';
-
-            treeNode[rootDatum.name] = {};
-            csv.forEach(function (row) {
-                var node = treeNode[rootDatum.name];
-                for (var col in row) {
-                    var type = row[col];
-                    var acronymMatches;
-                    var acronymRegex = /\((\w+)\)/g;
-
-                    //Ignore specific columns
-                    if (['metamaintype', 'metacolor'].indexOf(col) !== -1) break;
-
-                    if (!type) break;
-
-                    if (!(type in node)) {
-                        node[type] = {};
-                    }
+        d3.json(jsonUrl, function (oncotree_json) {
+            var rootNode = new UniqueTreeNodeDatum();
+            rootNode.name = 'Tissue';
+            rootNode.children = []
 
-                    if (!(type in uniqueTreeNode)) {
-                        uniqueTreeNode[type] = new UniqueTreeNodeDatum();
-                        uniqueTreeNode[type].name = type;
-
-                        acronymMatches = acronymRegex.exec(type);
-                        if (acronymMatches instanceof Array && acronymMatches[1]) {
-                            uniqueTreeNode[type].acronym = acronymMatches[1];
-                        }
-
-                        if(row.hasOwnProperty('metamaintype') && row.metamaintype !== '') {
-                            uniqueTreeNode[type].mainType = row.metamaintype;
-                        }
-                        else {
-                            uniqueTreeNode[type].mainType = 'Not Available';
-                        }
-
-
-                        if(row.hasOwnProperty('metacolor')) {
-                            uniqueTreeNode[type].color = row.metacolor;
-                        }
-
-                        if(row.hasOwnProperty('metanci')){
-                            uniqueTreeNode[type].nci = row.metanci;
-                        }
-
-                        if(row.hasOwnProperty('metaumls')){
-                            uniqueTreeNode[type].umls = row.metaumls;
-                        }
-
-                        if(row.hasOwnProperty('history')){
-                            uniqueTreeNode[type].history = row.history;
-                        }
-
-                    }
-                    node = node[type];
-                }
+            getOncotreeCodeKeysSortedByName(oncotree_json.data.TISSUE.children).forEach(function (code) {
+                var childData = oncotree_json.data.TISSUE.children[code];
+                // these nodes all belong at root of tree
```

Comment:
I think the TISSUE node is the root of the tree, so I would say "these nodes are all children of the TISSUE root node" or "these nodes are all tissue type nodes at depth 1" or similar.